megamek/src/megamek/common/options/IOption.java

```diff
@@ -29,27 +29,27 @@ public interface IOption extends IBasicOption, IOptionInfo {
     /**
      * Boolean option type
      */
-    public static final int BOOLEAN = 0;
+    int BOOLEAN = 0;
```

Comment:
Why make these non-static and modifiable?

Reply:
It's in an interface, so they're public static final automatically. IntelliJ warns about it and I tend to remove redundant clutter, but that's certainly subjective.
:grin:
![image](https://github.com/MegaMek/megamek/assets/17069663/0e3e89e8-78ae-4d46-aca1-f960af1adba9)

Reply:
Good to know.